src/plugins/saved_objects_management/server/routes/find.ts

```diff
@@ -24,6 +24,11 @@ export const registerFindRoute = (
   const searchOperatorSchema = schema.oneOf([schema.literal('OR'), schema.literal('AND')], {
     defaultValue: 'OR',
   });
+  const sortFieldSchema: Type<keyof v1.SavedObjectWithMetadata> = schema.oneOf([
+    schema.literal('created_at'),
+    schema.literal('updated_at'),
+    schema.literal('type'),
+  ]);
```

Comment:
This is probably the riskiest change to introduce here. The sort columns we define for the front end are pretty much fixed to these values but it is possible for columns to be registered dynamically and attempt to sort by a nested `attribute.*` field. This seems very unlikely and it does not look like this feature is being used by any other Kibana code.

Reply:
Seems reasonable to me. In fact, I like the approach that we have pre-defined options for sorting. We do, however, have to consider the case where a type may not have these fields, since we don't have a default option to sort on here. 

What do you think about defaulting to `type` if the other fields are missing? It's unlikely they can be though, so probably not needed.